React/training/src/app/components/Inputs/styles.module.scss

```diff
@@ -0,0 +1,54 @@
+.error {
+  border-color: #db2828;
+  display: inline-block;
+  line-height: 1;
+  vertical-align: baseline;
+  margin: 0 .14285714em;
+  background-color: #db2828;
+  background-image: none;
+  padding: .5833em .833em;
+  color: #fff;
+  text-transform: none;
+  font-weight: 700;
+  border: 0 solid transparent;
+  border-radius: .28571429rem;
+  -webkit-transition: background .1s ease;
+  transition: background .1s ease;
+  font-size: 0.7rem;
+  margin-bottom: 1rem;
+}
+
+.input {
+  padding: 0 1rem;
+  
+  input {
```

Comment:
I will not check css. But you have to create classes in order to be more specific. try to avoid this kind of css styles. Always use `clases`